geps/gep-713.md

```diff
@@ -525,13 +791,251 @@ type RouteRule struct {
+
+If we apply the default policy from earlier to the namespace:
+```yaml
+apiVersion: networking.example.io/v1alpha1
+kind: RetryOnPolicy
+metadata:
+  name: retryon5xx
+  namespace: appns
+spec:
+  defaults:
+    retryOn:
+      - "501"
+      - "502"
+      - "503"
+  targetRef:
+    kind: Namespace
+    group: ""
+    name: appns
+```
+
+Then this will have the same effect as applying that Policy to every Gateway in
+the `default` namespace - namely that every HTTPRoute that attaches to every
+Gateway will have its `retryOn` field set to `501`, `502`, `503`, _if_ there is no
+other setting in the HTTPRoute itself.
+
+With two layers in the hierarchy, we have a more complicated set of interactions
+possible.
+
+Let's look at some tables for a particular HTTPRoute, assuming that it does _not_
+configure the `retryOn` field, for various types of Policy at different levels.
+
+#### Overrides interacting with defaults for RetryOnPolicy, empty list in HTTPRoute
+
+||None|Namespace override|Gateway override|HTTPRoute override|
+|----|-----|-----|----|----|
+|No default|Empty list|Namespace override| Gateway override Policy| HTTPRoute override|
+|Namespace default| Namespace default| Namespace override | Gateway override | HTTPRoute override |
+|Gateway default| Gateway default | Namespace override | Gateway override | HTTPRoute override |
+|HTTPRoute default| HTTPRoute default | Namespace override | Gateway override | HTTPRoute override|
+
+#### Overrides interacting with other overrides for RetryOnPolicy, empty list in HTTPRoute
+||No override|Namespace override|Gateway override|HTTPRoute override|
+|----|-----|-----|----|----|
+|No override|Empty list|Namespace override| Gateway override| HTTPRoute override|
+|Namespace override| Namespace override| Namespace override<br />first created wins<br />otherwise first alphabetically | Namespace override | Namespace override |
+|Gateway override| Gateway override | Namespace override | Gateway override<br />first created wins<br />otherwise first alphabetically | Gateway override |
+|HTTPRoute override| HTTPRoute override | Namespace override | Gateway override | HTTPRoute override<br />first created wins<br />otherwise first alphabetically|
+
+#### Defaults interacting with other defaults for RetryOnPolicy, empty list in HTTPRoute
+||No default|Namespace default|Gateway default|HTTPRoute default|
+|----|-----|-----|----|----|
+|No default|Empty list|Namespace default| Gateway default| HTTPRoute default|
+|Namespace default| Namespace default| Namespace default<br />first created wins<br />otherwise first alphabetically | Gateway default | HTTPRoute default |
+|Gateway default| Gateway default | Gateway default | Gateway default<br />first created wins<br />otherwise first alphabetically | HTTPRoute default |
+|HTTPRoute default| HTTPRoute default | HTTPRoute default | HTTPRoute default | HTTPRoute default<br />first created wins<br />otherwise first alphabetically|
```

Comment:
Maybe just me, but I find these tables pretty confusing. https://deploy-preview-1565--kubernetes-sigs-gateway-api.netlify.app/references/policy-attachment/. The values in the cells seem to largely be just repeating the headers?

Reply:
That's kind of the idea, they're intended to be like truth tables representing the various interactions that make it clear that the `override` value always wins. I thought it might be better to be saying this in more than one way.

Reply:
I updated the truth tables with labels for rows and columns, hopefully that makes it a bit clearer.